tests/tasks/setup_candlepin_requirements.yml

```diff
@@ -0,0 +1,6 @@
+---
+collections:
+  - name: ansible.posix
+  - name: community.general
+  - name: community.mysql
+  - name: community.postgresql
```

Comment:
mysql and postgresql?

Reply:
the "candlepin" role has tasks for both; even though we configure to not depploy mariadb, it's easier to pull both those collections than working around in funky ways